src/converse-chatview.js

```diff
@@ -1018,6 +1026,25 @@ converse.plugins.add('converse-chatview', {
                 return this;
             },
 
+            async inject (ev) {
+                if (!_converse.debug) {
+                    return this;
+                }
+                const message_handler = _converse.chatboxes.onMessage.bind(_converse.chatboxes);
```

Comment:
You don't have to assign to `message_handler`, just call `_converse.chatboxes.onMessage` directly further down below.